Detectors/FIT/FV0/simulation/src/Digitizer.cxx

```diff
@@ -249,8 +253,8 @@ void Digitizer::storeBC(const BCCache& bc,
   triggers.setTriggers(isMinBias, isMinBiasInner, isMinBiasOuter, isHighMult, isDummy, nStored, totalChargeAllRing);
   digitsBC.emplace_back(first, nStored, bc, triggers);
   digitsTrig.emplace_back(bc, isMinBias, isMinBiasInner, isMinBiasOuter, isHighMult, isDummy);
-  int nBC = digitsBC.size();
-  for (const auto& lbl : bc.labels) {
+  size_t const nBC = digitsBC.size();
```

Comment:
As an additional fix of the bug with labels indexing, could you move L256 to a place before L254, so that the nBC is stored before incrementing the number of elements in digitsBC?

Reply:
Done. I've moved it to the beginning of this method where a similar statement (variable `first`).

Reply:
Thanks!